virttest/utils_net.py

```
@@ -4796,38 +4840,83 @@ def create_linux_bridge_tmux(
     else:
         cmd = "ip link add %s type bridge" % linux_bridge_name
     return utils_misc.cmd_status_output(
```

Ruff (0.13.1)

Function call with `shell=True` parameter identified, security issue

(S604)